hmda-analytics/src/main/scala/hmda/analytics/HmdaAnalyticsApp.scala

```diff
@@ -227,27 +214,27 @@ object HmdaAnalyticsApp extends App with TransmittalSheetComponent with LarCompo
-              case Period(2024, Some("Q1")) => transmittalSheetRepository2024Q1.deleteByLeiAndQuarter(lei = ts.lei)
-              case Period(2024, Some("Q2")) => transmittalSheetRepository2024Q2.deleteByLeiAndQuarter(lei = ts.lei)
-              case Period(2024, Some("Q3")) => transmittalSheetRepository2024Q3.deleteByLeiAndQuarter(lei = ts.lei)
+              case Period(2018, None) => YearlyTransmittalSheetRepositoryWrapper("2018").deleteByLei(ts.lei)
+              case Period(2019, None) => YearlyTransmittalSheetRepositoryWrapper("2019").deleteByLei(ts.lei)
+              case Period(2020, Some("Q1")) => QuarterlyTransmittalSheetRepositoryWrapper("2020", "Q1").deleteByLei(ts.lei)
+              case Period(2020, Some("Q2")) => QuarterlyTransmittalSheetRepositoryWrapper("2020", "Q2").deleteByLei(ts.lei)
+              case Period(2020, Some("Q3")) => QuarterlyTransmittalSheetRepositoryWrapper("2020", "Q3").deleteByLei(ts.lei)
+              case Period(2020, None) => YearlyTransmittalSheetRepositoryWrapper("2020").deleteByLei(ts.lei)
+              case Period(2021, None) => YearlyTransmittalSheetRepositoryWrapper("2021").deleteByLei(ts.lei)
+              case Period(2022, None) => YearlyTransmittalSheetRepositoryWrapper("2022").deleteByLei(ts.lei)
+              case Period(2023, None) => YearlyTransmittalSheetRepositoryWrapper("2023").deleteByLei(ts.lei)
+              case Period(2021, Some("Q1")) => QuarterlyTransmittalSheetRepositoryWrapper("2021", "Q1").deleteByLei(ts.lei)
+              case Period(2021, Some("Q2")) => QuarterlyTransmittalSheetRepositoryWrapper("2021", "Q2").deleteByLei(ts.lei)
+              case Period(2021, Some("Q3")) => QuarterlyTransmittalSheetRepositoryWrapper("2021", "Q3").deleteByLei(ts.lei)
+              case Period(2022, Some("Q1")) => QuarterlyTransmittalSheetRepositoryWrapper("2022", "Q1").deleteByLei(ts.lei)
+              case Period(2022, Some("Q2")) => QuarterlyTransmittalSheetRepositoryWrapper("2022", "Q2").deleteByLei(ts.lei)
+              case Period(2022, Some("Q3")) => QuarterlyTransmittalSheetRepositoryWrapper("2022", "Q3").deleteByLei(ts.lei)
+              case Period(2023, Some("Q1")) => QuarterlyTransmittalSheetRepositoryWrapper("2023", "Q1").deleteByLei(ts.lei)
+              case Period(2023, Some("Q2")) => QuarterlyTransmittalSheetRepositoryWrapper("2023", "Q2").deleteByLei(ts.lei)
+              case Period(2023, Some("Q3")) => QuarterlyTransmittalSheetRepositoryWrapper("2023", "Q3").deleteByLei(ts.lei)
+              case Period(2024, Some("Q1")) => QuarterlyTransmittalSheetRepositoryWrapper("2024", "Q1").deleteByLei(ts.lei)
+              case Period(2024, Some("Q2")) => QuarterlyTransmittalSheetRepositoryWrapper("2024", "Q2").deleteByLei(ts.lei)
+              case Period(2024, Some("Q3")) => QuarterlyTransmittalSheetRepositoryWrapper("2024", "Q3").deleteByLei(ts.lei)
```

Comment:
Don't we want to replace the deleteByLeiAndQuarter invocations with invocations of new deleteByLeiAndQuarter functions?

Reply:
We don't have to, and given the above comment, we shouldn't! Will fix